flict/__main__.py

```diff
@@ -175,15 +178,19 @@ def parse():
     parser_li = subparsers.add_parser(
         'list', help='list supported licenses')
     parser_li.set_defaults(which="list", func=list_licenses)
-    parser_li.add_argument('-r', '--relicensing',
-                           dest='list_relicensing',
-                           action='store_true',
-                           help='List relicensing information')
+#    parser_li.add_argument('-r', '--relicensing',
+#                           dest='list_relicensing',
+#                           action='store_true',
+#                           help='List relicensing information')
```

Comment:
If the options above are not needed, they should be removed. We should not introduce more dead code.